backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -63,4 +63,18 @@ class EakApplicationMutationService {
             ApplicationRepository.delete(applicationId, context)
         }
     }
+
+    @GraphQLDescription("Verifies an application verification")
+    fun verifyApplicationVerification(
```

Comment:
I feel like we should adjust the naming... But I don't really have a better one off the top of my head :D
... Maybe `VerifyOrRejectApplication` ?

Reply:
I was also struggling with this and considered creating two separate endpoints. But it seems like an unnecessary overhead. I'll use your proposition.